tests/unittests/Foundation/NSPointerArrayTests.mm

```diff
@@ -61,9 +55,12 @@ @implementation NSPointerArraySubClass
     ASSERT_EQ_MSG(array.count, 3, @"should have 3 items in the array.");
 }
 
-TEST(NSPointerArray, Subclassing) {
+// Note: On the reference platform when a class that does not support sub-classing is sub-classed, the parent class is
```

Comment:
I was looking at WindowsOnly tests and they seem to be mostly testing internal/deprecated methods that can never be tested on the reference platform. I would like to keep tests that do not match the reference platform under OSX_DISABLED_TEST so we are are aware of the difference and theoritically they are not "WindowsOnly" tests. Let us talk today in our standup

---
In reply to: [73437159](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73437159) [](ancestors = 73437159)